src/plugins/signoff/components.js

```diff
@@ -19,6 +19,44 @@ import AdminLink from "../../components/AdminLink";
 import { ProgressBar, ProgressStep } from "./ProgressBar.js";
 
 
+function isMember(groupkey, source, sessionState, bucketState) {
+  const {serverInfo: {user={}, capabilities}} = sessionState;
+  if (!user.id) {
+    return false;
+  }
+  const {signer={}} = capabilities;
+  const {[groupkey]: defaultGroupName} = signer;
```

Comment:
We could even do this in one line without intermediary `capabilities` and `signer` variables :) `#PerlFTW`